### ISSUE TITLE
NTR: Oxalate high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8262]
New term request Oxalate high in blood (elthat_100628030838):

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000567
- label: Elevated circulating oxalate concentration
- parents: [Dicarboxylic acidemia (HP:0040145)](https://hpo.jax.org/app/browse/term/HP:0040145)
- pmid: 
- def: The concentration of oxalate in the blood circulation is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8262
- synonyms:
